laravel/app/Models/Attending.php

```diff
@@ -19,4 +19,14 @@ class Attending extends Model
      * @var bool
      */
     public $timestamps = false;
+
```

Comment:
afaik, attending's write EvaluationData (should have a relationship)

Reply:
Fixed in next push!